### DISCUSSION TITLE
How to move to SD card?

### DISCUSSION THREAD

**Discussion by jiannazzone** [moraroy/NonSteamLaunchersDecky#21] [Q&A]
Thanks for making this incredibly handy tool! I'm trying to move a launcher to the SD card, but I can't find any way to do that. I've checked in Decky Launcher, Steam Deck settings, and on the app shortcut in my library. Thanks!

**Comment by moraroy** [comment]
@jiannazzone this option is only on the desktop version for now, i no longer have an SD card because mine broke :/ so i couldnt really test. If you use the desktop version and can test it and confirm it still works, i could try to add it in the plugin
